CONTRIBUTING.adoc

```diff
@@ -2,45 +2,135 @@
 :toc: left
```

Comment:
I've taken the liberty to shrink the README, and moved all development related stuff to CONTRIBUTING
Make a Development section and moved it right below the Introduction.
My line of reasoning was: If I want to contribute, I do want a short Introduction to the project, and then I just want to  know: "How to I build this?"